src/Search/Api/SearchController.php

```diff
@@ -4,7 +4,6 @@
 
 use DateTimeImmutable;
 use DateTimeZone;
-use Doctrine\Common\Cache\Cache;
```

Comment:
`doctrine/cache` can probably be removed from `composer.json`?

Reply:
It seems used to cache annotations: https://github.com/elifesciences/search/blob/800e57dc392694ca7763e6174963552f70a5df58/src/Search/Kernel.php#L166-L180